README.rst

```diff
@@ -80,6 +80,7 @@ Alternatively, provide AWS credentials using the settings below. This is useful
     AWS_SES_ACCESS_KEY_ID = 'my_access_key...'
     AWS_SES_SECRET_ACCESS_KEY = 'my_secret...'
     AWS_SES_REGION = 'us-west-2'
+    AWS_SES_CONFIGURATION_SET_NAME = 'my_configuration_set'
```

Comment:
Can you please provide a brief description about this setting in the paragraph above this code snippet? Perhaps with a link to https://docs.aws.amazon.com/ses/latest/DeveloperGuide/using-configuration-sets.html. I was unaware that configurations sets were directly related to Amazon SES until I Googled.